### DISCUSSION TITLE
Idea: Right click tree scene to quick open add-child popup?

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#9978] [Editor]
Idea: Right click empty space in tree scene left window to quick open add-child popup?

Thoughts?

**Comment by theraot** [comment]
This might work.

The only confusion part would be which node is the parent. Is it the selected `Node` or is it the root?

Keep in mind that there could be no selected `Node`, one selected Node or multiple.

**Reply by shak2** [reply]
Make the functionality of the right click exactly like clicking the "add child" button is now.

So:

If nothing selected - then add to root.

If something selected - then add as child to that.

(I think this is how it works, please verify before implementing!)
